jest.config.js

```diff
@@ -10,9 +11,8 @@ module.exports = {
       statements: 90,
     },
   },
-  modulePaths: ['<rootDir>'],
```

Comment:
I didn't need to make this and the `roots` change to get coverage working, we just don't need to set those configs here. `rootDir` is good enough.